synapse/rest/media/v1/upload_resource.py

```diff
@@ -56,7 +56,7 @@ def render_OPTIONS(self, request):
         if content_length is None:
             raise SynapseError(msg="Request must specify a Content-Length", code=400)
         if int(content_length) > self.max_upload_size:
-            raise SynapseError(msg="Upload request body is too large", code=413)
+            raise SynapseError(msg="Upload request body is too large", code=413,  errcode=Codes.TOO_LARGE)
```

Comment:
```suggestion
            raise SynapseError(
                msg="Upload request body is too large",
                code=413,
                errcode=Codes.TOO_LARGE,
            )
```

Linter's complaining again :confused: